src/utils/app.ts

```diff
@@ -45,10 +45,10 @@ export class AppBuilder {
     this.code.line("const app = new App();");
 
     this.config.stacks.forEach((stack) => {
+      this.code.line(`//  TODO: Add stack name
+//   e.g. { stack: "SomeStack" }`);
       this.code.line(
-        `new ${stack.name.pascal}(app, "${stack.name.pascal}", { app: "${
-          this.config.appName.kebab
-        }"${
+        `new ${stack.name.pascal}(app, "${stack.name.pascal}", {${
```

Comment:
Can we add the stack prop here instead of the TODO? I think something like this might work?

```suggestion
        `new ${stack.name.pascal}(app, "${stack.name.pascal}", { stack: "${
          stack.name.kebab
        }"${
```